qdrant_client/qdrant_fastembed.py

```diff
@@ -998,3 +1049,28 @@ def _embed_document(self, document: models.Document, is_query: bool = False) ->
             return embedding
         else:
             raise ValueError(f"{model_name} is not among supported models")
+
+    def _embed_image(self, image: models.Image) -> NumericVector:
+        """Embed an image using the specified embedding model
+
+        Args:
+            image: Image to embed
+
+        Returns:
+            NumericVector: Image's embedding
+
+        Raises:
+            ValueError: If model is not supported
+        """
+        model_name = image.model
+        if model_name in _IMAGE_EMBEDDING_MODELS:
+            embedding_model_inst = self._get_or_init_image_model(
+                model_name=model_name, **(image.options or {})
+            )
+            image_data = base64.b64decode(image.image)
```

Comment:
It does seem like a lot of overhead and not really a user-friendly interface, but I understand why it is done like this (I assume, cause we want local remove work the same)